channels/in.m3u

```diff
@@ -9,8 +9,8 @@ https://6n3yogbnd9ok-hls-live.5centscdn.com/threetamil/d0dbe915091d400bd8ee7f27f
 http://103.199.161.254/Content/7smusic/Live/Channel(7smusic)/index.m3u8
 #EXTINF:-1 tvg-id="9XM.in" tvg-name="9XM" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/fD7wLka.jpg" group-title="Music",9XM (480p)
 https://d2q8p4pe5spbak.cloudfront.net/bpk-tv/9XM/9XM.isml/index.m3u8
-#EXTINF:-1 tvg-id="ANEWSJATTTVBhangraFlava.in" tvg-name="A NEWSJATT TV- Bhangra Flava" tvg-country="IN" tvg-language="Malayalam" tvg-logo="" group-title="Music",A NEWSJATT TV- Bhangra Flava (720p)
-http://newsjatt.camdvr.org:1935/newsjatt/myStream/playlist.m3u8
+#EXTINF:-1 tvg-id="ANEWSDONSTVBhangraFlava.in" tvg-name="A NEWSDONS TV- Bhangra Flava" tvg-country="IN" tvg-language="Punjabi" tvg-logo="" group-title="Music",A NEWSDONS TV- Bhangra Flava (720p)
+http://tiny.cc/jattvlink
```

Comment:
This looks like a totally different stream. I would rather you add as a new stream than replace an existing.

Reply:
The funny thing is that why does it need to go through a tiny.cc link when you simply can give the stream link ?